runner/src/server/plugins/engine/pageControllers/PageControllerBase.ts

```diff
@@ -305,6 +306,68 @@ export class PageControllerBase {
     return request.yar.get("lang");
   }
 
+  calculateRouteState(page, completeState, state = {}) {
+    //Store our current routeState outside of our recursive method where it will be appended during the recursion
+    this.routeState = state;
```

Comment:
you are passing state as a parameter each time, so I don't think you need to use this.routeState?

Reply:
No longer needed. Thanks.